src/quo2/components/buttons/predictive_keyboard/style.cljs

```diff
@@ -0,0 +1,11 @@
+(ns quo2.components.buttons.predictive-keyboard.style)
+
+(defn wrapper
+  [type]
+  {:width              "100%"
+   :height             48
+   :justify-content    :center
+   :padding-horizontal (if (= type :words) 0 20)})
+
+(def word-list {:align-items :center :padding-horizontal 20})
+
```

Comment:
The files in this PR have an extra empty line. In `status-mobile` it's recommended to configure our editors to remove trailing new lines at the end of files and to automatically add a line break at the end of files (if one doesn't exist yet). Just so you never have to worry about this again :)